[filtron.sh] make filtron rule file configurable

return42 opened the pull request:
To select a different file with filtron rules, set environment

    FILTRON_RULES_TEMPLATE

the default is

    utils/templates/etc/filtron/rules.json

The installation is done by the new function install_rules() which offers a
multiple choice in case of collisions (known from searx.sh install setup).

update: a local test is not easy, you can do it in a LXC but I tested it on my instance.

return42 committed:
[filtron.sh] make filtron rule file configurable

To select a different file with filtron rules, set environment

    FILTRON_RULES_TEMPLATE

the default is

    utils/templates/etc/filtron/rules.json

The installation is done by the new function install_rules() which offers a
multiple choice in case of collisions (known from searx.sh install setup).

Signed-off-by: Markus Heiser <markus.heiser@darmarit.de>

dalf commented on utils/filtron.sh:
```diff
@@ -303,6 +302,47 @@ go get -v -u github.com/asciimoo/filtron
+            info_msg "install origin rules"
+            cp "${FILTRON_RULES_TEMPLATE}" "${FILTRON_RULES}"
+            ;;
+        "start interactiv shell")
```
small typo: `interactive`

return42 replied:
Oops .. fixed in 666bf1d

return42 committed:
[fix] typo: 'interactiv' --> 'interactive'

Reported-by: https://github.com/searxng/searxng/pull/38#pullrequestreview-646940531
Signed-off-by: Markus Heiser <markus.heiser@darmarit.de>
